app/views/shared/_side-nav.html.erb

```diff
@@ -8,18 +8,26 @@
     <ul class="main-menu">
         <li class="main-menu__item">
             <%= link_to_unless_current "Needs", root_path, class: "main-menu__link" do %>
+              <% if params[:user_id] || params[:status].eql?("to do") %>
+                <%= link_to "Needs", root_path, { class: "main-menu__current", style: "text-decoration: none" } %>
+              <% else %>
                 <span class="main-menu__current">Needs</span>
+              <% end %>
             <% end %>
 
             <ul class="main-menu__submenu">
                 <li class="main-menu__submenu-item">
-                    <%= link_to_unless_current "Assigned to me", root_path(:mine => true), class: "main-menu__submenu-link" do %>
-                        <span class="main-menu__submenu-current">Assigned to me</span> 
-                    <% end %>
+                  <% if current_page?(root_path) && params[:user_id] %>
```

Comment:
Wouldn't `link_to_unless_current` work with `?user_id` set? Seems like we're re-implementing its logic here, no?

Reply:
Yes, `link_to_unless_current` can work with `user_id`. But when we have more params for example `?user_id=<va>&status=<val>` we will lose `link_to_unless_current` behaviour.